Alignment/OfflineValidation/macros/trackSplitPlot.C

```diff
@@ -1663,795 +1918,729 @@ void makePlots(TString file,TString directory,Bool_t matrix[xsize][ysize])
-        if (pull)
-        {
-            tree->SetBranchAddress(sigma1variable,&sigma1);
-            tree->SetBranchAddress(sigma2variable,&sigma2);
-        }
-        if (relative && pull)
-            tree->SetBranchAddress(sigmaorgvariable,&sigmaorg);
-
-        for (Int_t i = 0; i<length; i++)
-        {
-            tree->GetEntry(i);
-            if (var == "runNumber" || var.BeginsWith("nHits"))
-                x = xint;
-            if (var == "runNumber")
-                runNumber = x;
-            if (var == "phi" && x >= pi)
-                x -= 2*pi;
-            if (var == "phi" && x <= -pi)
-                x += 2*pi;
-            if ((runNumber < minrun && runNumber > 1) || (runNumber > maxrun && maxrun > 0)) continue;
-
-            totallength++;
-
-            Double_t error;
-            if (relative && pull)
-                error = sqrt((sigma1/rel)*(sigma1/rel) + (sigma2/rel)*(sigma2/rel) + (sigmaorg*x/(rel*rel))*(sigmaorg*x/(rel*rel)));
-            else
-                error = sqrt(sigma1 * sigma1 + sigma2 * sigma2);   // = 1 if axis == 'x' && !pull
-                                                                   // = sqrt(2) if axis == 'y' && !pull, so that you get the error in 1 track
-                                                                   //       when you divide by it
-            x /= (rel * error);
-            if (!std::isfinite(x))  //e.g. in data with no pixels, the error occasionally comes out to be NaN
-                continue;           //Filling a histogram with NaN is irrelevant, but here it would cause the whole result to be NaN
-
-            if (what == Minimum && x < result)
-                result = x;
-            if (what == Maximum && x > result)
-                result = x;
-            xvect.push_back(x);
-            if (var.BeginsWith("nHits"))
-            {
-                x = xint2;
-                if (what == Minimum && x < result)
-                    result = x;
-                if (what == Maximum && x > result)
-                    result = x;
-                xvect.push_back(x);
-            }
-        }
-        delete f;         //automatically closes the file
+    if (relative && pull)
+      tree->SetBranchAddress(sigmaorgvariable, &sigmaorg);
+
+    for (Int_t i = 0; i < length; i++) {
+      tree->GetEntry(i);
+      if (var == "runNumber" || var.BeginsWith("nHits"))
+        x = xint;
+      if (var == "runNumber")
+        runNumber = x;
+      if (var == "phi" && x >= pi)
+        x -= 2 * pi;
+      if (var == "phi" && x <= -pi)
+        x += 2 * pi;
+      if ((runNumber < minrun && runNumber > 1) || (runNumber > maxrun && maxrun > 0))
+        continue;
+
+      totallength++;
+
+      Double_t error;
+      if (relative && pull)
+        error = sqrt((sigma1 / rel) * (sigma1 / rel) + (sigma2 / rel) * (sigma2 / rel) +
+                     (sigmaorg * x / (rel * rel)) * (sigmaorg * x / (rel * rel)));
```

Comment:
Same doubt as before: as it is repeated here it must be correct, but please confirm

Reply:
see https://github.com/cms-sw/cmssw/pull/43650/files/5947b05c24a5bcb9c4f39e2449afc8c615455c1c#r1441563349.